src/ViewTrait.php

```diff
@@ -189,6 +189,16 @@ public function withLocale(string $locale): static
         return $new;
     }
 
+    /**
+     * Get the specified locale code.
+     *
+     * @return string The locale code.
+     */
+    public function getLocale(): string
+    {
+        return $this->localeState->getLocale();
```

Comment:
Where's `localeState`?

Reply:
Good question.

It is inside `View` and `WebView`
https://github.com/yiisoft/view/blob/9547650ab106e8357613b75e38ac22a989d9f85c/src/WebView.php#L39-L45

And `state` also
https://github.com/yiisoft/view/blob/9547650ab106e8357613b75e38ac22a989d9f85c/src/ViewTrait.php#L228-L231

This task for a new issue.

Reply:
> This task for a new issue.

What is task?

`ViewTrait` is internal class that used in `View` and `WebView` only.

Reply:
> > This task for a new issue.
> 
> What is task?
> 
> `ViewTrait` is internal trait that used in `View` and `WebView` only.

Move `localeState` and `state` inside `ViewTrait`